src/main/java/uk/gov/hmcts/reform/sscs/service/DocumentPlaceholderService.java

```diff
@@ -73,27 +75,29 @@ private void setDwpAddress(Map<String, Object> placeholders, SscsCaseData caseDa
             && nonNull(caseData.getAppeal().getMrnDetails())
             && nonNull(caseData.getAppeal().getMrnDetails().getDwpIssuingOffice())) {
 
-            final Optional<DwpAddress> optionalAddress = dwpAddressLookup.lookup(
+            final DwpAddress dwpAddress = dwpAddressLookup.lookup(
```

Comment:
I think if this is empty then we should be throwing an exception here otherwise the letter is going to have a blank address

Reply:
Actually, looks like you already do this further down. Guess ignore above comment.